### PR TITLE
Remove unhandled except

### PR DESCRIPTION
**Pull request by Fizzadar**

### CODE REVIEW

File: queue_util/consumer.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def run_forever(self, wait_timeout_seconds=None, **kwargs):
                 #
                 break
 
-            except:
+            except Exception:
```
Comment (sujaymansingh) [inline queue_util/consumer.py:172]: Is there much of a difference between `except` and `except Exception`? They'll both 'catch' all exceptions.

Comment (Fizzadar) [reply queue_util/consumer.py:172]: 👋 hey @sujaymansingh - no massive difference, but there are some slight nuances: https://moserei.de/2017/09/20/python-bare-except.html

Comment (bobbyrenwick) [reply queue_util/consumer.py:172]: Thanks @Fizzadar, good to know you can now kill these bad boys with a Ctrl-C straight to the heart.

Comment (sujaymansingh) [reply queue_util/consumer.py:172]: 👍 That makes sense @Fizzadar , I didn't know about that! (In fact, I would put an explicit `except KeyboardInterrupt` statement and exit from there, something that is probably spread across the code that you guys have to deal with now :))

**Review by sujaymansingh — APPROVED** [review]

**Review by bobbyrenwick — APPROVED** [review]

**Comment by gchazot** [comment]
This is superseeded by #30

**Comment by gchazot** [comment]
release #30 in 3.2.0

**Pull request closed without merging**